test/web3/walletClient.spec.ts

```diff
@@ -83,4 +91,52 @@ describe.skip('WalletClient', () => {
       expect(baseAccount?.address).toEqual(secondAccount.address);
     });
   });
+
+  describe('getBaseAccount', () => {
+    test('should return the base account', async () => {
+      const fetchedBaseAccount = await web3Client.wallet().getBaseAccount();
+      expect(fetchedBaseAccount).not.toBeNull();
+      console.log('fetchedBaseAccount', fetchedBaseAccount);
+      expect(fetchedBaseAccount?.address).toEqual(baseAccount.address);
+    });
+
+    test('should return null if base account is not set', async () => {
+      await web3Client.wallet().cleanWallet();
+      const fetchedBaseAccount = await web3Client.wallet().getBaseAccount();
+      expect(fetchedBaseAccount).toBeNull();
+    });
+  });
+
+  describe('getWalletAccounts', () => {
+    test('should return all accounts in the wallet', async () => {
+      const accounts = [
+        await WalletClient.walletGenerateNewAccount(),
+        await WalletClient.walletGenerateNewAccount(),
+        await WalletClient.walletGenerateNewAccount(),
+      ];
+
+      await web3Client.wallet().addAccountsToWallet(accounts);
+      const walletAccounts = await web3Client.wallet().getWalletAccounts();
+      expect(walletAccounts.length).toBe(4);
```

Comment:
Why is it 4 and not 3 ?

Reply:
there also is the baseAccount

Reply:
Can you add a comment to explain that for the next dev who check the unit test ?